package.json

```diff
@@ -30,13 +30,14 @@
     "postcss": "postcss src/**/*.css --base src --dir es",
     "publish-patch": "npm version patch | xargs -I {} sh -c 'git push origin {}'",
     "postpublish-patch": "git push origin main",
-    "predeploy": "DEPLOYMENT_ENV=production npm run build-static",
-    "deploy": "gh-pages -d imageviewer -o github"
+    "prepare": "DEPLOYMENT_ENV=production npm run build",
+    "predeploy": "DEPLOYMENT_ENV=production npm run dev",
+    "deploy": "gh-pages -d imageviewer -t true"
   },
   "author": "Megan Riel-Mehan",
   "license": "ISC",
   "dependencies": {
-    "@aics/volume-viewer": "^2.4.0",
+    "@aics/volume-viewer": "AllenInstitute/volume-viewer#feature/load-zarr",
```

Comment:
this line will be updated prior to merging, after volume-viewer branch is merged and it gets a version bump